resources/party.ts

```diff
@@ -15,6 +17,103 @@ const emptyRoleToPartyNames = () => {
   };
 };
 
+const roleLocalized: Record<Role, LocaleText> = {
+  tank: { en: 'tank', ja: 'タンク', cn: '坦克' },
```

Comment:
Is it possible to put these fields on separate lines for the find translations script, e.g.

```
const roleLocalized: Record<Role, LocaleText> = {
  tank: {
    en: 'tank',
    ja: 'タンク',
    cn: '坦克',
  },
```